services/minespace-web/src/styles/components/SideNavigation.scss

```diff
@@ -0,0 +1,80 @@
+.sidebar-wrapper {
+    .ant-layout-sider {
+        background-color: $card-bg;
+        padding: 0 8px;
+
+        &.sider-fixed {
+            margin-top: $layout-header-height;
+            overflow: auto;
+            height: 100%;
+            position: fixed;
+            top: 0;
+            left: 0;
+            bottom: 0;
+        }
+
+        .ant-layout-sider-children {
+            display: flex;
+            flex-direction: column;
+            gap: $spacing-vertical;
+        }
+
+        .sidebar-header {
+            background-color: $card-bg;
+            padding: 12px;
+            display: flex;
+            flex-direction: column;
+            gap: 12px;
+        }
+
+        .ant-menu {
+            background-color: #F7F8FA;
+        }
+
+        .ant-menu-vertical {
+            border-right: none;
+
+            .ant-menu-item {
+                border-radius: 8px;
+                font-size: 14px;
+                margin: 0;
+                line-height: 44px;
+                height: 44px;
+                color: $font-colour;
+
+                &.ant-menu-item-selected {
+                    background-color: "#D9EAF7";
+                }
+            }
+        }
+    }
+
+    .ant-layout-content {
+        padding: 0 $spacing-vertical;
+
+        .dashboard-add-button {
+            margin-top: $spacing-vertical;
+            z-index: 1;
+            display: inline;
+            float: right;
+        }
+    }
+
+    h1 {
```

Comment:
Maybe this is the future of what we want h1-h3 to look like, but localized for now because I'm not sure.